frontend/src/pages/FAQ.vue

```diff
@@ -2,6 +2,10 @@
   <q-page padding class="text-center">
     <h2 class="page-title">Frequently Asked Questions</h2>
 
+      <div class="text-caption chinese-faq"> Languages: EN | 
+        <a href="https://comet-friend-aa9.notion.site/Umbra-073a96aabac44dc185a116efb8f4c138" target="_blank">中文</a>
```

Comment:
Just needs our custom `hyperlink` class so the colors look correct in both light mode and dark mode, but otherwise this PR lgtm!
```suggestion
        <a class="hyperlink" href="https://comet-friend-aa9.notion.site/Umbra-073a96aabac44dc185a116efb8f4c138" target="_blank">中文</a>
```